### ISSUE TITLE
Provider SSM seems limited to 10 parameters per map

### ISSUE THREAD

**Issue by popul** [tellerops/teller#276]
With teller 2.0.5, when i create a provider SSM with one map like this:
```yaml
  ssm:
    kind: ssm
    maps:
    - id: nonprod
      path: /project
```
And i put more than 10 parameters prefixed by /project in SSM parameter store

`teller show` and `teller sh` return only 10 parameters.

It should return all parameters, shouldn't it ?

**Comment by jondot** [comment]
yes, looks like a limit in pagination, we will resolve it in next version

**Comment by jondot** [comment]
should be resolved with `2.0.6`
